CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.2.0-beta.2 (Unreleased)
+
+- Add `az_iot_provisioning_client_get_request_payload()` to create MQTT payload bodies during Device Provisioning.
```

Comment:
@danewalton  - I just checked GitHub the handle CaptainPreposition is still open if you ever want to rebrand yourself:).

This could go either way, since the "during" could be "during the entire provisiniong process" - so like topic build/payload-build/MQTT-send/MQTT-receive.  "For" would probably be marginally better.  

At this point though... the change is in `main` and its in a changelog.  While there's no enforcement this is append-only file, they tend to be like that in practice except for really bad mistakes.  My inclination is to leave this as-is.

Reply:
Yea ignore me haha no worries. Less change the better here.